packages/product-json-to-csv/test/map-headers.spec.js

```diff
@@ -19,7 +19,7 @@ describe('mapHeaders', () => {
         'name.en',
         'description.en',
         'slug.en',
-        'attribute',
+        'attributes',
```

Comment:
I'm wondering why we're removing singular `attribute` here? Is it never used and was just a typo?

Reply:
According to the [docs](https://docs.commercetools.com/http-api-projects-products.html#productvariant), it is `attributes`.
`attribute` was declared in the code but was never called or used except in the test.
Guess it may be a typo error.

Reply:
`attributes` is the right key name.